release/publish.yaml

```diff
@@ -196,10 +196,11 @@ spec:
             do
               crane cp ${IMAGE_WITH_SHA} ${REGION}.${IMAGE_WITHOUT_SHA_AND_TAG}:$TAG
             done
+            echo ${REGION}.$IMAGE_WITH_SHA, >> $(results.IMAGES.path)
```

Comment:
thanks @priyawadhwa for explaining 😍.

In that case, instead of having `echo` in two places, the original `echo` can be moved out of `else`:

```shell
        for REGION in ${REGIONS}
        do
          if [[ "$(params.releaseAsLatest)" == "true" ]]
          then
            for TAG in "latest" $(params.versionTag)
            do
              crane cp ${IMAGE_WITH_SHA} ${REGION}.${IMAGE_WITHOUT_SHA_AND_TAG}:$TAG
            done
          else
            TAG="$(params.versionTag)"
            crane cp ${IMAGE_WITH_SHA} ${REGION}.${IMAGE_WITHOUT_SHA_AND_TAG}:$TAG
          fi
          echo ${REGION}.$IMAGE_WITH_SHA, >> $(results.IMAGES.path)
        done
```

Reply:
thank you! i've updated it :)